targets/unity-v2/source/Authentication/PlayFabAuthService.cs

```diff
@@ -131,67 +121,79 @@ public void ClearRememberMe()
         }
 
         /// <summary>
-        /// Kick off the authentication process by specific authtype.
+        /// Kick off the authentication process by specific authType.
         /// </summary>
         /// <param name="authType"></param>
-        public void Authenticate(AuthTypes authType)
+        /// <param name="authKeys"></param>
+        public void Authenticate(AuthTypes authType, AuthKeys authKeys = null)
         {
             AuthType = authType;
-            Authenticate();
+            Authenticate(authKeys);
         }
 
-        public void Authenticate()
+        public void Authenticate(AuthKeys authKeys = null)
         {
-            var authType = AuthType;
-
-            if (authType == AuthTypes.None)
+            if (AuthType == AuthTypes.None)
             {
-                if(OnDisplayAuthentication != null)
+                if (OnDisplayAuthentication != null)
                     OnDisplayAuthentication.Invoke();
+
+                return;
+            }
+
+            if (RememberMe && !string.IsNullOrEmpty(RememberMeId))
+            {
+                PlayFabClientAPI.LoginWithCustomID(new LoginWithCustomIDRequest
+                {
+                    TitleId = PlayFabSettings.TitleId,
+                    CustomId = RememberMeId,
+                    InfoRequestParameters = InfoRequestParams,
+                    CreateAccount = true
+                }, InvokeLoginSuccess, InvokePlayFabError);
                 return;
             }
+            
+            var auth = _authStrategies[AuthType];
```

Comment:
If AuthType is invalid, then this will just throw an exception rather than assigning auth to null, so the null check 2 lines down doesn't really serve the purpose you expect it to.

Go ahead and just remove the whole ```if (auth == null) { log, return }``` block below, because it will never go there.